snippets/capabilities-table.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: SNIPPET
+
+The following table describes the `baselineCapabilitySet` values.
+
+.Cluster capabilities `baselineCapabilitySet` values description
+[cols=".^4,.^6a",options="header"]
+|===
+|Value|Description
+
+|`vCurrent`
+|Specify when you want to automatically add new capabilities as they become recommended.
+
+|`v4.11`
+|Specify when you want the capabilities recommended in {product-title} 4.11 and not automatically enable capabilities, which might be introduced in later versions.
+
+|`v4.12`
+|Specify when you want the capabilities recommended in {product-title} 4.12 and not automatically enable capabilities, which might be introduced in later versions.
+
```

Comment:
Suggest to list the content of each set `vCurrent`, `v4.11`, `v4.12`, so that user know which operators are enabled when setting `baselineCapabilitySet`, and better to understand if any additional capabilities need to be added in `additionalEnabledCapabilities` based on `baselineCapabilitySet` setting.
https://github.com/openshift/api/blob/master/config/v1/types_cluster_version.go#L344-L380

Reply:
Good idea, I've listed them here